### DISCUSSION TITLE
zoom image

### DISCUSSION THREAD

**Discussion by kalle07** [Uminosachi/sd-webui-inpaint-anything#122] [Q&A]
hey,

may can you doo the same like the original inpaint ... that you can zoom in and out the picture ?

 ;)

**Comment by kalle07** [comment]
sry   "S" and "R"   ;)

**Reply by Uminosachi** [reply]
The output images are displayed in a gallery format, and clicking on them shows an enlarged view. The "S" and "R" keys are not supported.
